networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/forks/GossipForkManager.java

```diff
@@ -246,6 +249,23 @@ private <T> void publishMessage(
                     slot));
   }
 
+  private <T> SafeFuture<Void> publishMessageWithFeedback(
+      final UInt64 slot,
+      final T message,
+      final String type,
+      final BiFunction<GossipForkSubscriptions, T, SafeFuture<Void>> publisher) {
+    final Optional<GossipForkSubscriptions> gossipForkSubscriptions =
+        getSubscriptionActiveAtSlot(slot).filter(this::isActive);
+
+    if (gossipForkSubscriptions.isEmpty()) {
+      LOG.warn(
```

Comment:
If we expand it over block proposing we may need supress here. Not needed now, but just ideas aloud